server/service/orbit.go

```diff
@@ -170,14 +163,16 @@ func getOrbitConfigEndpoint(ctx context.Context, request interface{}, svc fleet.
 }
 
 func (svc *Service) GetOrbitConfig(ctx context.Context) (fleet.OrbitConfig, error) {
+	const pendingScriptMaxAge = 24 * time.Hour // TODO(mna): how long should we try to get a script executed?
```

Comment:
@noahtalerman Any idea at what point we should start ignoring old script execution requests (scripts that were attempted to be sent to a host for execution, but never got executed yet eg. because the host is offline)? Could be a month, I guess, but a year would seem way too long (i.e. no one would likely think about going back to look at the results)?

Reply:
@mna is this for running a script async? (`POST /run`) Or is it for both sync and async?

Reply:
@noahtalerman it's unrelated to whether we ask it to run sync or async, it's to get the list of scripts that should be sent to the host for execution (so, regardless of if it's sync or async, we do have to send it to the host at some point).

So the question is, if the host is offline for a long time, at what point do we stop sending it a very old script execution request? If the script execution was requested more than 30 days ago? Currently I set it at 1 day ago, but just to set it to something and start the discussion.

Reply:
@mna ok, so if I'm understanding correctly, Orbit asks the Fleet server for a list of scripts that should be executed. 

If the host is offline, the Fleet server might have "queued" up several scripts. So we're deciding at what point do we remove an old script from the queue?

I think in this iteration we don't have a "pending" state (coming next sprint) because we only support running the script on hosts that are online. 

So off the top of my head I think we can stop sending old script requests after 1 day. 

That said, I'm not positive we have any UX to cover the case in which we stop sending the old script request...

I scheduled a call w/ you, @marko-lisica, and I to discuss (in 45 mins).

Reply:
Update following the call:

* Ok to use 1 day for now to stop sending old script requests
* Need to add a validation in the `/run/sync` API endpoint to check that the targeted host is currently detected as `online`

I'll add the validation as part of this PR.